test/core/end2end/fuzzers/network_input.cc

```diff
@@ -307,4 +314,142 @@ Duration ScheduleReads(
       });
   return Duration::Milliseconds(delay + 2);
 }
+
+namespace {
+
+void ReadForever(std::shared_ptr<EventEngine::Endpoint> ep) {
+  bool finished;
```

Comment:
It seems like this variable actually indicates when we're *not* finished.  Maybe call it something like `reading`?

Reply:
will rename in the next pr (submitting now to unblock builds)